dashboard/app/views/levels/editors/_ailab.html.haml

```diff
@@ -1,4 +1,5 @@
 = render partial: 'levels/editors/fields/instructions_area', locals: {f: f}
+= render partial: 'levels/editors/fields/ailab_dynamic_instructions', locals: {f: f}
```

Comment:
Can you move ailab_dynamic_instructions into the instructions area? Preferably between these two lines so its closer to the other instructions?

```
  = render partial: 'levels/editors/fields/instructions', locals: {f: f} unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
  = render partial: 'levels/editors/fields/help_and_tips', locals: {f: f} if @level.show_help_and_tips_in_level_editor?
```

Reply:
Done!